### DISCUSSION TITLE
My inverter has 6 PV strings, only 4 are visible

### DISCUSSION THREAD

**Discussion by owelie** [hultenvp/solis-sensor#372] [Q&A]
My inverter has 6 PV strings. However, the Solis-sensor only shows 1-4. Is this something that can be fixed simply by adding PV 5 and 6 sections in the const.py file, or will additional coding be needed?

Anybody that can point me in the right direction? 

Thanks

**Comment by poligor84** [comment]
Greetings. I have the same problem.
Did you manage to solve it?
Thank you.
